histomicsui/web_client/templates/layout/header.pug

```diff
@@ -17,3 +17,7 @@ nav.navbar.navbar-default(style=`color: ${brandColor}; background-color: ${banne
         li.h-image-menu-wrapper.dropdown
         li.h-analyses-wrapper.dropdown
         li.h-current-user-wrapper.dropdown
+        if helpURL
+          li
+            a.h-help-button(role='button', title=helpTooltip, href=helpURL, target='_blank')
```

Comment:
For accessibility reasons this should probably be a button instead of link tag but will defer to @manthey  about whether it should be changed or not.